### ISSUE TITLE
NTR: 4-hydroxy-2-oxoglutarate high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8867]
New term request 4-hydroxy-2-oxoglutarate high in urine (elthat_140808083745): HOG; 4-hydroxyoxoglutarate

**Comment by pnrobinson** [comment]
@MickeySegal
what is the clinical context?
I cannot find this on chebi or OMIM.

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK316514/
"increased 4-hydroxy-2-oxoglutarate (HOG)"

elsewhere in the article the 2 is omitted.

**Comment by pnrobinson** [comment]
@MickeySegal 
I am interpreting it as this: https://hmdb.ca/metabolites/HMDB0060466

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001009
- label: Increased urine 4-hydroxy-2-oxoglutarate level
- parents: [Organic aciduria (HP:0001992)](https://hpo.jax.org/app/browse/term/HP:0001992)
- pmid: PMID:26401545
- def: The amount of 4-hydroxy-2-oxoglutarate in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8867
- synonyms:
